osu.Game/GameModes/Menu/Button.cs

```diff
@@ -128,7 +128,7 @@ protected override bool OnHover(InputState state)
                 LoopDelay = duration * 2
             });
 
-            icon.Transforms.Add(new TransformPosition(Clock)
+            icon.Transforms.Add(new TransformPosition()

```

Comment:
You left the parens for some constructors and removed them for others. Let's remove all?